resources/js/Components/AppHead.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+import { Head } from '@inertiajs/vue3'
+
+defineProps<{
+  title: string
+  description?: string
+}>()
```

Comment:
You can set a default value in this way

```suggestion
withDefault(defineProps<{
  title: string
  description?: string
}>(), { description: 'Default description'  })
```